spring-mvc/src/main/java/com/kodemon/springmvc/controllers/FightController.java

```diff
@@ -196,7 +196,7 @@ else if (mode.equals("catch")) {
 
         boolean fightResult = fightFacade.fightWildPokemon(user, wildPokemon, mode_);
         String wildPokemonName = wildPokemon.getName().getName();
-        PokemonDTO userPokemon = user.getPokemons().get(0);
+        PokemonDTO userPokemon = user.getActivePokemons().get(0);
         String userPokemonName = userPokemon.getName().getName();
```

Comment:
Not really pretty :smile: maybe we should consider renaming `PokemonName` field in `PokemonDTO` and `Pokemon` to `pokemonName` instead of `name` :smile:. Also, it can lead to NPE or other exception.